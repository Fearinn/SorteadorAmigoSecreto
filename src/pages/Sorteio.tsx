import Card from "../components/Card";
import Sorteador from "../components/Sorteador";
const Sorteio = () => {
 
  return (
    <Card>
     <Sorteador/>
    </Card>
  );
};

export default Sorteio;
