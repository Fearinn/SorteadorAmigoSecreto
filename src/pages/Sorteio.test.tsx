import { render } from "@testing-library/react";
import { RecoilRoot } from "recoil";
import Sorteio from "./Sorteio";

const mockNavegacao = jest.fn();

jest.mock("react-router-dom", () => {
  return {
    useNavigate: () => mockNavegacao,
  };
});

describe("página de sorteio", () => {
  test("a renderização ocorre corretamente", () => {
    const {container}  = render(
      <RecoilRoot>
        <Sorteio />
      </RecoilRoot>
    );

    expect(container).toMatchSnapshot();
  });
});