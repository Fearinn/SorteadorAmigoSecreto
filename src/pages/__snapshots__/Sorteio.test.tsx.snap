// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`página de sorteio a renderização ocorre corretamente 1`] = `
<div>
  <div
    class="card"
  >
    <section
      class="section"
    >
      <h1
        class="titulo"
      />
      <p
        class="amigo"
        role="alert"
      />
      <button
        class="botao"
      >
        Sortear
      </button>
      <div
        class="imagem"
      />
    </section>
  </div>
</div>
`;
